core/controllers/ingest.py

```diff
@@ -152,10 +152,14 @@ def parse_body(body: dict) -> tuple[str, int, dict | None, bool]:
     reporting_round = body.get("reporting_round")
     auth = parse_auth(body)
     do_load = body.get("do_load", True)  # defaults to True, if False then do not load to database
+    submitting_account_id = body.get("submitting_account_id", None)
+    submitting_user_email = body.get("submitting_user_email", None)
 
     # Set these values for reporting sentry metrics via `core.metrics:capture_ingest_metrics`
     g.fund_name = fund
     g.reporting_round = reporting_round
+    g.submitting_account_id = submitting_account_id
+    g.submitting_user_email = submitting_user_email
```

Comment:
I think we can avoid using `g` here by returning this metadata directly, and passing it through to the `populate_db` function.

I think it's nice to keep the use of `g` as small as possible because it's a global namespace and has all of the traditional issues with global namespaces. This one's reasonably inoffensive because we're only using these values in one place, but I also think that we can just pass them through the function call chain quite easily as well (unlike the other things that we're sticking on `g` here).

Reply:
Removed the new fields from g object and passed them into functions.